### ISSUE TITLE
does not return a response or status

### ISSUE THREAD

**Issue by shortthefomo** [Transia-RnD/hooks-toolkit-ts#46]
https://github.com/Transia-RnD/hooks-toolkit-ts/blob/2634cd7da64f4782b601d42b695c37a872f1d73d/src/setHooks.ts#L55

this should return if the hook was set successfully or not.

**Comment by dangell7** [comment]
You're absolutely right. It is hard fail but I think I'l adjusting that so I will add the response as users may want that for whatever reason.
